sunbird-platform-core/common-util/src/main/java/org/sunbird/common/request/orgvalidator/AssignKeyValidator.java

```diff
@@ -0,0 +1,92 @@
+package org.sunbird.common.request.orgvalidator;
+
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.request.BaseRequestValidator;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+
+import java.text.MessageFormat;
+import java.util.List;
+
+/**
+ * this class is used to validate the request of the OrgAssignKeys Controller
+ * @author anmolgupta
+ */
+public class AssignKeyValidator extends BaseRequestValidator {
```

Comment:
AssignKeyValidator to keyManagementValidator

Reply:
Acknowledged and Executed